bzt/bza.py

```diff
@@ -741,20 +745,24 @@ def get_addr(self):
 
         proxy_info = response['result']
         if proxy_info:
+            self.log.info('Proxy already exists')
```

Comment:
Maybe implement automatic delete of proxy in this case? Would be much more user-friendly to just work.

Reply:
Not sure it's the best idea to delete the current proxy automatically, I'm giving the user the option to delete it by himself